cores/arduino/stm32/usb/cdc/usbd_cdc_if.c

```diff
@@ -212,6 +222,7 @@ static int8_t USBD_CDC_Receive(uint8_t *Buf, uint32_t *Len)
 
 static int8_t USBD_CDC_Transferred(void)
 {
+  transmitStart = 0;

```

Comment:
Hum, I guess it should be 
```suggestion
  transmitStart = HAL_GetTick();
```

Reply:
There was a line missing from the commit quashing. I added it.

Reply:
the line is correct, this call indicates end of a transfer so we stop the timer.
the timer would be reinitialized if there is something to transmit inside CDC_continue_transmit

Reply:
Ah right👍